bundles/org.openhab.binding.bosesoundtouch/src/main/java/org/openhab/binding/bosesoundtouch/internal/ContentItem.java

```diff
@@ -27,30 +28,23 @@
  * @author Christian Niessner - Initial contribution
  * @author Thomas Traunbauer - Initial contribution
  */
+@NonNullByDefault
 public class ContentItem {
 
-    private String source;
-    private String sourceAccount;
-    private String location;
-    private boolean presetable;
-    private String itemName;
-    private int presetID;
-    private String containerArt;
+    private String source = "";
+    private @Nullable String sourceAccount;
+    private @Nullable String location;
+    private boolean presetable = false;
+    private @Nullable String itemName;
+    private int presetID = 0;
+    private @Nullable String containerArt;
     @Expose
-    private final Map<String, String> additionalAttributes;
+    private final Map<String, String> additionalAttributes = new HashMap<>();
 
     /**
      * Creates a new instance of this class
      */
     public ContentItem() {
-        source = "";
-        sourceAccount = null;
-        location = null;
-        presetable = false;
-        itemName = null;
-        presetID = 0;
-        containerArt = null;
-        additionalAttributes = new HashMap<>();
     }
```

Comment:
Since it no longer has an implementation, it is equal to the default constructor, so you can probably remove the whole constructor.

Reply:
Done